src/platform_impl/web/web_sys/canvas/pointer_handler.rs

```diff
@@ -105,11 +115,129 @@ impl PointerHandler {
         ));
     }
 
+    pub fn on_touch_move<F>(&mut self, canvas_common: &super::Common, mut handler: F)
+    where
+        F: 'static + FnMut(i32, PhysicalPosition<f64>, Force),
+    {
+        let canvas = canvas_common.raw.clone();
+        self.on_touch_move = Some(canvas_common.add_event(
+            "pointermove",
+            move |event: PointerEvent| {
+                if event.pointer_type() != "touch" {
+                    return;
+                }
+
+                handler(
+                    event.pointer_id(),
+                    touch_physical_position(&event, &canvas),
+                    Force::Calibrated {
+                        force: event.pressure() as f64,
+                        max_possible_force: 1.0,
+                        altitude_angle: None, // I don't have any idea how to calculate this from tilt_x and tilt_y.
+                    },
```

Comment:
This should probably be `Force::Normalized`, since the pressure ranges from 0 to 1 and there's no 'average' pressure.

Reply:
Thanks, I resolved this.